DOI.js

```diff
@@ -191,6 +199,11 @@ var testCases = [
 		"type": "web",
 		"url": "https://en.wikipedia.org/wiki/Template_talk:Doi",
 		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://zotero.org/?d=10.7208/chicago/9780226924632.001.0001",
+		"items": "multiple"
 	}
```

Comment:
Can you also add https://onlinelibrary.wiley.com/doi/abs/10.1002/(SICI)1099-1360(199711)6:6%3C320::AID-MCDA164%3E3.0.CO;2-2 as a test case?